src/main/java/core/basesyntax/Forming.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Forming {

```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-use-verbs-for-classinterface-names

And more understandable to name it "Drowable"